frontend/src/metabase/dashboard/components/DashboardHeader/buttons/AddSectionButton.tsx

```diff
@@ -31,25 +30,25 @@ export const AddSectionButton = () => {
   return (
     <Menu position="bottom-end">
       <Menu.Target>
-        <span>
-          <Tooltip label={t`Add section`}>
-            <DashboardHeaderButton aria-label={t`Add section`}>
-              <Icon name="section" size={18} />
-            </DashboardHeaderButton>
-          </Tooltip>
-        </span>
+        <ToolbarButton
+          tooltipLabel={t`Add section`}
+          aria-label={t`Add section`}
+          icon="section"
+        />
       </Menu.Target>
       <Menu.Dropdown miw="100px">
         <Flex direction="column" align="center" gap="md" p="12px">
           {layoutOptions.map(layout => (
-            <SectionMenuItem
+            <Menu.Item
+              className={CS.bgBrandHover}
```

Comment:
This doesn't work as expected likely because how we're styling `[data-hovered]`
![image](https://github.com/user-attachments/assets/13df97ca-1afd-4241-a36f-b00e8db0926f)

1. Toggle hover in the devtool
2. Actual hover
3. non-hover

Reply:
one idea if you want to override it is to append `[data-hovered]` to the `bgBrandHover`, but I'm not sure if that's the best course going forward since we seem to only theme Mantine component in `metabase/ui/X.styled.ts`

Reply:
already tried it  - the CSS class is just not as specific as the styles in mantine :upside\_down\_face:

Reply:
interestingly, for [Match all filter popover styles with dark theme](https://github.com/metabase/metabase/issues/45723#top), I only theme Mantine components in those `*.styled.ts` files.